### ISSUE TITLE
Feature request: Replace OpenGL interface with ANGLE using Metal

### ISSUE THREAD

**Issue by kode54** [MinecraftMachina/multimc-macos-arm64#23]
Since clearly Apple's OpenGL implementation is broken, and ANGLE is actually coming along nicely, maybe it's time to investigate replacing the OpenGL.framework interface with ANGLE, wrapping on Metal.

**Comment by kode54** [comment]
Or possibly even better, this, if it turns out to be usable:

https://www.reddit.com/r/opengl/comments/qtxgph/its_alive_opengl_45_on_mac_os_metal/

**Comment by ViRb3** [comment]
Happy to add this to ManyMC as soon as somebody actually gets it working with Minecraft :)

**Comment by rinOfTheStars** [comment]
yeah getting this working is going to be hard, and also a long-term project for everyone trying to get apple silicon running Minecraft in full
ill investigate ANGLE soonish, and im already keeping tabs on MGL, ill let everyone here know if any progress occurs

**Comment by rinOfTheStars** [comment]
hm, yeah ANGLE wont work
its OpenGL ES, which is not the same as OpenGL (ES is a web standard, and LWJGL3.x already supports it's Metal wrappers on macOS)

**Comment by kode54** [comment]
Yeah, I was meaning more a full OpenGL wrapper, if that ever materializes. I wasn't expecting any big changes overnight, I was just hoping that topic I linked to would eventually publish results that proved to be usable.

**Comment by kode54** [comment]
This looks to be the thing: https://github.com/openglonmetal/MGL

**Comment by r58Playz** [comment]
See https://github.com/openglonmetal/MGL/issues/55
